piksi_gps/piksi_gps/piksi_gps_pub.py

```diff
@@ -68,6 +74,17 @@ def __init__(self):
 
 
     def piksi_log_callback(self, signal, *args, **kwargs):
+        '''
```

Comment:
Method docstring extended summary should give more detail on what the method does, such as what types of sbp messages are actually handled by the callback. Currently provides no info on what required datapoints and respective ros topics are about.